smokey/src/main/java/frc/robot/subsystems/DrivetrainSubsystem.java

```diff
@@ -418,7 +420,12 @@ public void periodic() {
     else { // SwerveDriveMode.NORMAL_DRIVING
```

Comment:
I was originally confused about this comment as it seemed wrong ... the SwerveDriveMode and the SwerveDriveRotationMode are very similar ... can we combine them?

Reply:
They are not combinable, since they are independent options.  We can rename the SwerveDriveRotationMode to SwerveDriveCenterOfRotation.  It could be CenterOfRobot and FrontOfRobot?